src/main/java/seedu/uninurse/model/util/SampleDataUtil.java

```diff
@@ -67,9 +69,8 @@ public static Set<Tag> getTagSet(String... strings) {
     /**
      * Returns a {@code TaskList} containing the list of strings given.
      */
-    public static TaskList getTaskList(String... strings) {
-        ArrayList<Task> taskArrayList = Arrays.stream(strings)
-                .map(Task::new).collect(Collectors.toCollection(ArrayList::new));
+    public static TaskList getTaskList(Task... strings) {
+        ArrayList<Task> taskArrayList = Arrays.stream(strings).collect(Collectors.toCollection(ArrayList::new));
```

Comment:
Maybe rename strings to taskStrings for clarity?

Reply:
Sure